hubmap/portal/tests.py

```diff
@@ -0,0 +1,37 @@
+from django.test import RequestFactory, TestCase
+
+# Create your tests here.
+from .views import details, entities, index, listall
+
+
+class PortalViewsTestCase(TestCase):
+    def setUp(self) -> None:
+        self.factory = RequestFactory()
+        self.request = self.factory.request()
+        self.entity_type = 'donor'
+        self.hubmap_id = 4
+        pass
+
+    def test_home_page(self):
+        response = index(self.request)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsNotNone(response.content)
+        pass
```

Comment:
Remove extra `pass`

Reply:
Perhaps there's confusion about the meaning of `pass`? It doesn't mean "pass" the test... It's really "pass"-through: See [docs](https://docs.python.org/3/tutorial/controlflow.html?highlight=pass#pass-statements)